app/frontend/src/features/auth/AuthPage.tsx

```diff
@@ -47,61 +62,145 @@ const useStyles = makeStyles((theme) => ({
+      margin: "0 auto",
+      width: "960px",
+    },
+  },
+  desktopNavigation: {
+    display: "flex",
+  },
+  introduction: {
+    color: "#ffffff",
+    textAlign: "left",
+    width: "742px",
+  },
+  loginLink: {
+    border: `1px solid ${theme.palette.primary.main}`,
+    borderRadius: "4px",
+    color: "#ffffff",
+    fontWeight: 500,
+    fontSize: "20px",
+    marginRight: theme.spacing(3),
+    padding: theme.spacing(1, 0),
+    textAlign: "center",
+    textDecoration: "none",
+    width: "88px",
   },
-  navigation: {
+  mobileNavigation: {
     display: "flex",
     justifyContent: "space-around",
-    marginTop: theme.spacing(3),
+    marginTop: theme.spacing(3, 0),
+  },
+  signupLink: {
+    backgroundColor: theme.palette.primary.main,
+    borderRadius: "4px",
+    color: "#ffffff",
+    fontWeight: 500,
+    fontSize: "20px",
+    padding: theme.spacing(1),
+    textAlign: "center",
+    textDecoration: "none",
+    width: "88px",
   },
   subtitle: {
     marginTop: theme.spacing(1),
+    [theme.breakpoints.up("md")]: {
+      display: "inline-block",
+      marginTop: theme.spacing(5),
+      position: "relative",
+    },
+  },
+  title: {
+    ...theme.typography.h1,
+    [theme.breakpoints.up("md")]: {
+      fontSize: "64px",
+      lineHeight: "74px",
+      textAlign: "left",
+    },
+  },
+  underline: {
+    borderTop: `5px solid ${theme.palette.primary.main}`,
+    boxShadow: "0px 4px 4px rgba(0, 0, 0, 0.25)",
+    left: theme.spacing(1),
+    position: "absolute",
+    width: "100%",
   },
-  title: theme.typography.h1,
 }));
 
 export default function AuthPage() {
   const classes = useStyles();
   const authClasses = useAuthStyles();
 
   return (
-    <Box className={classes.authPage}>
-      <Box className={classes.content}>
-        <Typography variant="h1">{INTRODUCTION}</Typography>
-        <Typography classes={{ root: classes.subtitle }}>
-          {COLLABORATIVE}
-        </Typography>
-        <Divider classes={{ root: authClasses.divider }} flexItem />
-        <Box className={classes.navigation}>
-          <Button
-            component={Link}
-            to={loginRoute}
-            classes={{
-              label: authClasses.buttonText,
-              root: `${authClasses.button} ${classes.button}`,
-            }}
-            color="secondary"
-          >
-            {LOGIN}
-          </Button>
-          <Button
-            component={Link}
-            to={signupRoute}
-            classes={{
-              label: authClasses.buttonText,
-              root: `${authClasses.button} ${classes.button}`,
-            }}
-            color="secondary"
-          >
-            {SIGN_UP}
-          </Button>
-        </Box>
-        <Box className={classes.aboutUs}>
-          <Typography className={classes.aboutUsLink} component={Link} to="#">
-            {ABOUT_US}
+    <div className={classes.authPage}>
+      <Hidden mdUp>
+        <div className={classes.content}>
```

Comment:
It would be better if this was a separate component, one for mobile, one for desktop to avoid confusion with the `Hidden`s. But in the interest of time, for now maybe just put a big comment `{/***** DESKTOP ******/}` and one for mobile